src/esp/bindings/SensorBindings.cpp

```diff
@@ -105,6 +105,10 @@ void initSensorBindings(py::module& m) {
   py::class_<VisualSensor, Magnum::SceneGraph::PyFeature<VisualSensor>, Sensor,
              Magnum::SceneGraph::PyFeatureHolder<VisualSensor>>(m,
                                                                 "VisualSensor")
+      .def_property_readonly(
+          "get_render_camera", &VisualSensor::getRenderCamera,
```

Comment:
Having a property is much nicer, thank you for doing that.  For properties, no `get_` tho, just `render_camera`